package.json

```diff
@@ -33,7 +33,7 @@
     "@hmcts/draft-store-client": "^1.1.1",
     "@hmcts/info-provider": "^1.0.0",
     "@hmcts/nodejs-healthcheck": "^1.4.5",
-    "@hmcts/nodejs-logging": "^2.2.0",
+    "@hmcts/nodejs-logging": "file:./../../hmcts/nodejs-logging",
```

Comment:
`yarn link @hmcts/nodejs-logging` is a better way of doing this, it doesn't change the package.json file instead it removes the directory in your node_modules and adds a symlink
`yarn unlink @hmcts/nodejs-logging && yarn add @hmcts/nodejs-logging` when you are done working locally on it

Reply:
done